### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by angolini**

### CODE REVIEW

File: source/reference-manual/linux/linux.rst [hunk 1]
```diff
@@ -11,7 +11,7 @@ select set of board support package layers to enable popular
 development boards.
 
 Though the `Linux kernel`_ and software used for
-any on of the microPlatform builds may contain out-of-tree patches or
+any one of the microPlatform builds may contain out-of-tree patches or
```
Comment (angolini) [inline source/reference-manual/linux/linux.rst:14]: I am really confused here. Any help is appreciated. For me it looks like a word s missing.

Comment (vanmaegima) [reply source/reference-manual/linux/linux.rst:14]: Maybe just leave it as "any of the..."?

Comment (ricardosalveti) [reply source/reference-manual/linux/linux.rst:14]: I think we can just remove on/one here, a let it as 'used for any of the microPlatform', what do you think?

**Comment by doanac** [comment]
Docs for da7e46e41a167b978a184407959fd17b41a4cf7d are browesable at: https://ci.foundries.io/projects/fio-docs/builds/773/docs/artifacts/html/index.html

**Review by MatthewCroughan — APPROVED** [review]
Thanks for this.

**Pull request merged**